lib/twilio/call.ts

```diff
@@ -628,15 +628,17 @@ class Call extends EventEmitter {
 
       this._pstream.addListener('hangup', this._onHangup);
 
+      this._onAnswerBind = this._onAnswer.bind(this);
+
       if (this._direction === Call.CallDirection.Incoming) {
         this._isAnswered = true;
-        this._pstream.on('answer', this._onAnswer.bind(this));
+        this._pstream.on('answer', this._onAnswerBind);
         this._mediaHandler.answerIncomingCall(this.parameters.CallSid, this._options.offerSdp,
           rtcConstraints, rtcConfiguration, onAnswer);
       } else {
         const params = Array.from(this.customParameters.entries()).map(pair =>
          `${encodeURIComponent(pair[0])}=${encodeURIComponent(pair[1])}`).join('&');
-        this._pstream.on('answer', this._onAnswer.bind(this));
+        this._pstream.on('answer', this._onAnswerBind);
```

Comment:
Can't you just turn the `_onAnswer` to an arrow function (same as the other class methods) to avoid this issue from happening?